src/igvfd/types/file_set.py

```diff
@@ -37,14 +37,18 @@ class FileSet(Item):
         'control_for': ('FileSet', 'control_file_sets')
     }
     embedded_with_frame = [
-        Path('award', include=['@id', 'component']),
+        Path('award', include=['@id', 'component', 'contact_pi']),
+        Path('award.contact_pi', include=['@id', 'title']),
         Path('lab', include=['@id', 'title']),
         Path('submitted_by', include=['@id', 'title']),
         Path('files', include=['@id', 'accession', 'aliases']),
         Path('control_for', include=['@id', 'accession', 'aliases']),
         Path('donors', include=['@id', 'accession', 'aliases', 'taxa']),
-        Path('samples', include=['@id', 'accession', 'aliases',
-             'sample_terms', 'summary', 'taxa']),]
+        Path('samples', include=['@id', 'accession', 'aliases', 'treatments', 'cell_fate_change_treatments',
```

Comment:
This is the embedding on file_sets for samples, I would have liked to avoid embedding some of these since they are not used by all file sets but they are embedded further in measurement set and there doesn't seem to be an (obvious) way to "expand" the sample properties to embed for just measurement set.